### PR TITLE
Send selected date to the voucher-history-email endpoint

### PR DESCRIPTION
**Pull request by ob6160**

### CODE REVIEW

File: src/pages/Account.vue [hunk 1]
```diff
@@ -83,18 +83,24 @@
...
         created: function () {
 
         },
         methods: {
+            recordSelect: function(event) {
+                this.selected = false;
+                // Default to requesting all.
+                this.selectedDate = event.target.id || null;
+            },
             onRequestSubmissionEmail : function(event) {
```
Comment (katjam) [inline src/pages/Account.vue]: Do we still need the event here?

Comment (ob6160) [reply src/pages/Account.vue]: nope!

**Review by katjam — COMMENTED** [review]
I think this is good to go now.. but maybe just check that the API expects null and not ''

File: src/pages/Account.vue [hunk 2]
```diff
@@ -83,18 +83,24 @@
                 voucherPayments: Store.trader.pendedVouchers,
                 errorMessage : Store.error,
                 goodFeedback : false,
-                selected : true
+                selected : true,
+                selectedDate: null
```
Comment (katjam) [inline src/pages/Account.vue:87]: I think it's fine - but can't remember what the API expects.

Comment (ob6160) [reply src/pages/Account.vue:87]: I just checked - it looks like null

Comment (katjam) [reply src/pages/Account.vue:87]: :+1:

**Review by katjam — APPROVED** [review]
Do this thing! :boat:

**Pull request merged**